pubspec.yaml

```diff
@@ -51,3 +51,10 @@ workiva:
   core_checks:
     version: 1
     react_boilerplate: disabled
+
+dependency_validator:
+  exclude:
+    - app/**
+    - tools/**
+  ignore:
+    - meta
```

Comment:
Based on the CI failure I think we need this in `tools/analyzer_plugin/pubspec.yaml` as well